lib/methods/get-community.js

```diff
@@ -0,0 +1,66 @@
+'use strict';
+
+// node core modules
+
+// 3rd party modules
+const _ = require('lodash');
+
+// internal modules
+const { omitDefaultRequestParams, constructError } = require('./utils');
+const { getCommunity: parser } = require('../response-parsers');
+
+const qsValidParameters = ['communityUuid'];
+
+/**
+ * Get all details for a community given its id.
+ *
+ * @param  {String}   communityUuid     URL parameter that is unique for every community.
+ * @param  {Object}   options           Additional information required by building http request.
+ * @param  {Function} callback          Called with `(err, community)`
+ */
+function getCommunity(communityUuid, options, callback) {
+  const { httpClient } = this;
+  if (!communityUuid) {
+    const error = constructError('{{ query.communityUuid }} must be defined in [get] request', 404);
```

Comment:
error message misleading, since there is no `query` argument to this method. communityUuid is a standalone arg